src/components/common/ServicesTable/ServiceTable.js

```diff
@@ -148,96 +148,6 @@ const ServicesTable = ({
         );
       },
     },
-    // {
```

Comment:
YES! Please, when you see dead code, remove it. It will still be in the commit history or if you're worried about losing the commented out code, save it to a snippet in your browser, or a gist on github, or even just a text document. Commented out code, has little to no value in a repo. It's just whitespace. 

Excellent removal here!